packages/ui/react-ui-form/src/components/Form/fields/MarkdownField.tsx

```diff
@@ -43,6 +43,11 @@ export const MarkdownField = ({
     adjustHeight();
   }, [getValue(), adjustHeight]);
 
+  const handleChange = useCallback(
```

Comment:
Simpler and more standard:
`useCallback<NonNullable<TextAreaProps['onChange']>>`

multiple places